developer_manual/core/ui-testing.rst

```diff
@@ -41,18 +41,25 @@ Set Up Test
 - Set the following environment variables:
 
   - ``SRV_HOST_NAME`` (the hostname where ownCloud runs)
+  - ``REMOTE_FED_SRV_HOST_NAME`` (alternative hostname for federation share tests, this should be an other IP/hostname of the same server)
```

Comment:
s/alternative hostname for federation share tests, this should be/An alternative hostname for federation share tests. This should be/g